src/components/widgets/BaseInfoWidget.vue

```diff
@@ -36,7 +36,7 @@
 							{{getNameByKey('blockHeight')}}
 						</div>
 						<div class="ex-item-value">
-							{{blockHeight}}
+							{{chainInfo.currentHeight}}
```

Comment:
I think it is better to move this part to comuted `chainInfo.currentHeight`. And put here only `{{chainHeight}}`